packages/nodes-base/credentials/PushoverApi.credentials.ts

```diff
@@ -23,9 +23,9 @@ export class PushoverApi implements ICredentialType {
 		requestOptions: IHttpRequestOptions,
 	): Promise<IHttpRequestOptions> {
 		if (requestOptions.method === 'GET') {
-			Object.assign(requestOptions.qs, { token: credentials.apiKey });
+			Object.assign(requestOptions.qs ?? {}, { token: credentials.apiKey });
```

Comment:
Moved to use a nullish check instead of `?? {}`, to remain closer to current behavior. `Object.assign(undefined, obj)` throws so we know the target is defined in these calls. Dug deeper and it looks like root cause is `Object.assign` overloads now [constraining the target param](https://github.com/microsoft/TypeScript/commit/6e62273fa1e7469b89b589667c2c233789c62176#diff-c4b87a3c28600f5d77bbc99ce5062acfa048f028216e9d0a96adcf937887644e) to be a subset of `{}`.